src/Factory/ApiProblemListenerFactory.php

```diff
@@ -4,15 +4,15 @@
 
 namespace Laminas\ApiTools\ApiProblem\Factory;
 
-use Interop\Container\ContainerInterface;
+use interop\container\containerinterface;
```

Comment:
Please undo this or use the PSR-11 container interface.

Reply:
@froschdesign done :)